drivers/hmis_csv_importer/app/jobs/hmis_csv_importer/cleanup/expire_base_job.rb

```diff
@@ -127,8 +124,8 @@ def process_model(model)
 
     private def populate_tmp_table_query(model, tmp_table_name)
       <<~SQL
-        INSERT INTO #{tmp_table_name} (source_id, batch_id)
-        SELECT id, #{log_id_field} from (#{relevant_id_query(model)}) as relevant_ids
+        INSERT INTO #{tmp_table_name} (source_id)
+        SELECT id FROM (#{relevant_id_query(model)}) as relevant_ids
```

Comment:
Pretty sure we still need the `log_id_field` for performance until we've added the index to the non-temporary table.

Reply:
where you able to see a difference in the total run time from adding the index? Benchmarking locally I didn't see much difference (1 mil rows)

Reply:
I haven't had a chance to benchmark. It definitely helped on the non-temp tables, but with lack of iops balance, nothing is fast in QA right now. Happy to try without, and add to the import/loader tables after the first round of deletes.